addons/mail/static/src/components/composer_suggested_partners/composer_suggested_partners.js

```diff
@@ -0,0 +1,43 @@
+odoo.define('mail/static/src/components/composer_suggested_partners/composer_suggested_partners.js', function (require) {
+    'use strict';
+    const { Component } = owl;
+    const useStore = require('mail/static/src/component_hooks/use_store/use_store.js');
+
+    class ComposerSuggestedPartners extends Component {
+        constructor(...args) {
+            super(...args);
+            useStore(props => {
+                const composer = this.env.models['mail.composer'].get(props.composerLocalId);
+                const suggestedPartners = this.composer.thread.suggestedPartners;
+                return {
+                    composer: composer ? composer.__state : undefined,
+                    thread: composer && composer.thread
+                        ? composer.thread.__state
+                        : undefined,
+                    suggestedPartners
+                };
+            });
+        }
+
+        async willStart() {
+            await this.composer.thread.getSuggestedPartners();
+        }
```

Comment:
We don't like async in components, we prefer to show it with incomplete information than delay its rendering.

It should automatically update itself when `getSuggestedPartners` is updating related store.